src/common/playlists/PlayList.jsx

```diff
@@ -12,13 +12,22 @@ import { useLocation } from 'react-router-dom';
 import { ParseQuery, QueryDBTranslator } from 'common/search/search-helper';
 import { getPlaysByFilter } from 'common/services/plays';
 
+const SORT_BY = ['Newest', 'Oldest', 'Most Liked', 'Random'];
+
 const PlayList = () => {
   const [randomPlay, setRandomPlay] = useState({});
   const [loading, setLoading] = useState();
   const [plays, setPlays] = useState();
   const [isFiltered, setIsFiltered] = useState(false);
+  const [sortBy, setSortBy] = useState(localStorage.getItem('sortByPlay') || 'Newest');
```

Comment:
please check if we have redux with persistence then use redux instead of saving this to localstorage directly

Reply:
`sortBy` is a simple state that will be used on `/plays` page only.
Why to go with **_central_** redux/context?

Reply:
localStorage is being used for [saving the preference](https://github.com/reactplay/react-play/issues/1159#issuecomment-1598037778) on page reload.

Reply:
Saying in such a way that this is not a good practice of calling localStorage.getItem directly in useState. We also had localstorage hooks in order to handle localstorage operation. Other than that its fine.